ConsoleApp/Program.cs

```diff
@@ -0,0 +1,211 @@
+﻿using System;
+using System.Threading;
+using System.Collections.Generic;
+using System.Linq;
+namespace ConsoleApp;
+
+    class Game
+    {
+        static readonly int x = 50;
+        static readonly int y = 16;
+
+        static Walls walls;

```

Comment:
code convention violated
```suggestion
        privtae static Walls _walls;
```